### PR TITLE
Add Kyverno policies to mutate `imagePullPolicy` and `terminationGracePeriodSeconds`

### PR DESCRIPTION
**Pull request by leo8a**
Add Kyverno policy examples, in the community collection, to configure and mutate resources.

Signed-off-by: Leo Ochoa <lochoa@redhat.com>

### CODE REVIEW

File: community/CM-Configuration-Management/policy-kyverno-container-tgps.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                          kinds:
+                            - Pod
+                      preconditions:
+                        - key: "{{`{request.object.spec.terminationGracePeriodSeconds}}`}}"
+                          operator: GreaterThan
+                          value: 50   # maximum tGPS allowed by admin
+                      mutate:
+                        patchStrategicMerge:
+                          spec:
+                            terminationGracePeriodSeconds: 50
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-kyverno-container-tgps.yaml:53]: Thanks for the PR.  Can you add the `PlacementBinding` and `PlacementRule` resources for these policies similar to how they are included for other policies?

File: community/CM-Configuration-Management/policy-kyverno-image-pull-policy.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                      match:
+                        resources:
+                          kinds:
+                            - Pod
+                      mutate:
+                        patchStrategicMerge:
+                          spec:
+                            containers:
+                              - (name): "?*"
+                                imagePullPolicy: Always
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-kyverno-image-pull-policy.yaml:52]: Same request here for the `PlacementBinding` and `PlacementRule` to make deploying the policy easier.  Thanks

**Review by gparvin — COMMENTED** [review]
Added request for placement resources in the policies.  Thanks for creating this pr and including fixes for some other issues.

File: community/CM-Configuration-Management/policy-kyverno-container-tgps.yaml (hunk 1 above)
Comment (leo8a) [reply community/CM-Configuration-Management/policy-kyverno-container-tgps.yaml:53]: sure, I just added it to all the contributed policies, thanks.

File: community/README.md [hunk 3]
```diff
@@ -115,6 +115,7 @@ Policy  | Description | Prerequisites
...
 [Policy to setup ODF](./CM-Configuration-Management/policy-odf.yaml) | Use this policy to install and configure the OpenShift Data Foundation | making it work would require e.g. on AWS m5 instances would be required. Requires OpenShift 4.6 or later.
 [Policy to install Kyverno](./CM-Configuration-Management/policy-install-kyverno.yaml) | Use this policy to install Kyverno | consult the following [link](https://github.com/kyverno) to get more information about Kyverno.
+[Kyverno config exclude resources](./CM-Configuration-Management/policy-kyverno-config-exclude-resources.yaml) | Use the Kyverno policy to exclude resources from certain processes for all constraints in the cluster | See the [resource-filters-in-Kyverno](https://kyverno.io/docs/installation/#resource-filters).
```
Comment (leo8a) [inline community/README.md:118]: Similar to [Gatekeeper](https://github.com/open-cluster-management/policy-collection/blob/main/community/CM-Configuration-Management/policy-gatekeeper-config-exclude-namespaces.yaml), being able to exclude specific resources looks like a good practice here. Hence, I added this policy to achieve so by patching the default Kverno configmap.

**Review by gparvin — APPROVED** [review]
Thanks for making the updates and for contributing these new policies.

**Pull request merged**